<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Embed</class>
 <widget class="QWidget" name="Embed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Embed</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>141</height>
           </size>
          </property>
          <property name="title">
           <string>Author</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Author URL</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>80</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Author Icon URL</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="authorInput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>351</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="authorURL">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>171</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="authorIconURL">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>100</y>
             <width>171</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>999</height>
           </size>
          </property>
          <property name="title">
           <string>Body</string>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="embedTitle">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>351</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="embedDescription">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>351</width>
             <height>91</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>57</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Title URL</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>200</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="embedURL">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>220</y>
             <width>221</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="colorInput">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>220</y>
             <width>41</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="selectColorButton">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>220</y>
             <width>71</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>121</height>
           </size>
          </property>
          <property name="title">
           <string>Fields</string>
          </property>
          <widget class="QPushButton" name="addFieldButton">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>30</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="editFieldButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>60</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="deleteFieldButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>90</y>
             <width>80</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QListWidget" name="fieldsList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>261</width>
             <height>81</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>141</height>
           </size>
          </property>
          <property name="title">
           <string>Images</string>
          </property>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Image</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="imageInput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>351</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Thumbnail</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="thumbnailInput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>351</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="maximumSize">
           <size>
            <width>371</width>
            <height>131</height>
           </size>
          </property>
          <property name="title">
           <string>Footer</string>
          </property>
          <widget class="QLineEdit" name="footerInput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>351</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>57</width>
             <height>14</height>
            </rect>
           </property>
           <property name="text">
            <string>Footer</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="timestampCheckbox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>TimeStamp</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>80</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Footer Icon URL</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="footerIconURL">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>100</y>
             <width>201</width>
             <height>22</height>
            </rect>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addEmbed">
      <property name="text">
       <string>Add Embed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
